pmd-java/src/main/resources/category/java/codestyle.xml

```diff
@@ -1351,6 +1351,40 @@ public class ClassInDefaultPackage {
         </example>
     </rule>
 
+    <rule name="NumericLiteralConvention"
```

Comment:
I think naming the rule "UseUnderscoresInNumericLiterals" may be more adapted, though it's open to discussion. Unless we want to add more power to the rule, e.g. enforce using always upper- or lowercase for the `d` or `f` suffix

Reply:
This certainly makes sense. I think for now we can go ahead with "UseUnderscoresInNumericLiterals" and later this can be renamed if more rules related to numeric literal convention is added.